homework/generate-sequence/generateSequence.hpp

```diff
@@ -2,6 +2,18 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+
+    if(count <= 0 ){
+        return{};
+    }
+    else{
+        std::vector<int> vec;
```

Comment:
Reserve memory for vector, when you know how many elements will be inserted. 
This will avoid the situation with resizing vector few times (which is always expensive).